docs/source/tutorials/hospital_admissions_model.qmd

```diff
@@ -179,11 +180,34 @@ I0 = InfectionInitializationProcess(
 
 # Generation interval and Rt
 gen_int = deterministic.DeterministicPMF(gen_int, name="gen_int")
-rtproc = process.RtRandomWalkProcess(
-    Rt0_dist=dist.TruncatedNormal(loc=1.2, scale=0.2, low=0),
-    Rt_transform=transformation.ExpTransform().inv,
-    Rt_rw_dist=dist.Normal(0, 0.025),
-)
+
+
+class MyRt(metaclass.RandomVariable):
+
+    def validate(self):
+        pass
+
+    def sample(self, n_timepoints: int, **kwargs) -> tuple:
+        sd_rt = numpyro.sample("Rt_random_walk_sd", dist.HalfNormal(0.025))
```

Comment:
Could we do this as RandomVariable? Even better if we could pass it as an argument to `MyRt()`.

Reply:
For a first tutorial with a user-defined RV I was thinking of keeping things a bit more concrete, but we can / should absolutely point out to the user that they can achieve additional configurability at the cost of additional abstraction when they built `RandomVariables` and models generally:

Some examples in the present context:

A not-much-more-generic case in which we always want a Half-Normal prior on the Normal random walk step s.d. but want to set a fixed scale value for that Half-Normal at class instantiation:

```python
class MyRt(RandomVariable):
   def __init__(self, rw_step_sd_prior_halfnormal_scale: float):
```

A not-much-more-generic-than-that case in which the Half-Normal prior's scale is a `RandomVariable`:

```python
class MyRt(RandomVariable):
   def __init__(self, rw_step_sd_prior_halfnormal_scale: RandomVariable):
```

A case in which we pass in a pre-instantiated prior distribution for the random walk s.d.: 

```python
class MyRt(RandomVariable):
   def __init__(self, rw_step_sd_prior: numpyro.distirbutions.Distribution):
```

A case in which we completely defer the question of how to determine the step s.d. and make it is own `RandomVariable`:

```python
class MyRt(RandomVariable):
   def __init__(self, rw_step_sd_rv: RandomVariable):
```

And finally, a case in which we completely defer the question of how to determine the step _at all_ and make _that_ its own `RandomVariable`:

```python
class MyRt(RandomVariable):
   def __init__(self, rw_step_rv: RandomVariable):
```

All of these design choices are possible. We should think about which if any we want to encourage users to make.

Reply:
It would make https://github.com/CDCgov/multisignal-epi-inference/issues/284 easier

Reply:
@dylanhmorris I think it is best practice to make all priors configurable at the highest level possible. When specifying a model, you should not have to rewrite a class just to change the prior it uses.

Reply:
I agree RE: production code but I also think that concreteness in examples / tutorials can help make them clearer. In any case separate PR but I agree that we should probably have a tutorial demonstrating what we believe to be a readable but also modular and re-configurable way of specifying a complete model.